shared/js/lockscreen_slide.js

```diff
@@ -343,12 +332,10 @@
     function lss_initializeCanvas() {
 
       this.overlay = document.getElementById(this.IDs.overlay);
-      this.area = document.getElementById(this.IDs.area);
```

Comment:
after this change following attempts to use this member will fail with: ``JavaScript Error: "TypeError: this.area is null"``